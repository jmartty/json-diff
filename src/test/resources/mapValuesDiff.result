JsonPrimitive diff, expected: "hola" but was "chau"
Stack[0]: Object["key1"]