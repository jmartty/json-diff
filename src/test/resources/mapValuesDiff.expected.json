{
  "key1": "hola",
  "key2": true
}